src/components/BookForm/BookForm.js

```diff
@@ -1,20 +1,41 @@
-const BookForm = () => (
-  <form>
-    <input type="text" />
-    <label htmlFor="category">
-      Choose a category:
-      <select name="category" id="category">
-        <option value="Action">Action</option>
-        <option value="Biography">Biography</option>
-        <option value="History">History</option>
-        <option value="Horror">Horror</option>
-        <option value="Kids">Kids</option>
-        <option value="Learning">Learning</option>
-        <option value="Sci-Fi">Sci-Fi</option>
-      </select>
-    </label>
-    <button type="submit">Submit</button>
-  </form>
-);
+import { useState } from 'react';
+import { CREATE_BOOK } from '../../actions/index';
+import store from '../../reducers/index';
+
+const BookForm = () => {
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('Action');
+  const [id, setId] = useState(4);
```

Comment:
- [ ] For the book Ids, use Math.random or use the uuid4 package to generate random ids.

Reply:
Resolved